### PR TITLE
fix styles so that tabpanel grabs the available space

### LINKED ISSUE #1466: Tabs header grabs complete vertical space
# 🐛 Bug Report 

Using the Tabs-Component the complet available space is not used for the content but for the header with the tab-items. This can be reproduced with the following snippet

```typescript
function Reproducer() {
  return (<Flex minHeight={400} minWidth={400} UNSAFE_style={{borderWidth: 1, borderStyle: 'solid', borderColor: 'black'}}>
    <Tabs>
      <Item title="Tab 1">
        <Content>
          <Text>Hello World</Text>
        </Content>
      </Item>
      <Item title="Tab 2">
        <Content>
          <Text>Hello World</Text>
        </Content>
      </Item>
    </Tabs>
  </Flex>);
}
```

yielding the following result

Bildschirmfoto 2021-01-14 um 18 09 50

## 🤔 Expected Behavior

I would expect that the content area takes all available space

## 😯 Current Behavior

The tab area takes all space available

## 💁 Possible Solution

The reason that the header takes the complete space is that it gets the following CSS-Selector applied

Bildschirmfoto 2021-01-14 um 19 29 45

Removing the three properties "flex-grow", "flex-shrink" and "flex-basis" from the node and adding "flex-grow" to the tabpanel would fix the problem.

Bildschirmfoto 2021-01-14 um 19 35 02

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |
| Browser          |
| Operating System |

### PR DESCRIPTION
**Pull request by tomsontom**
Closes #1466

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@adobe/spectrum-css-temp/components/tabs/index.css [hunk 1]
```diff
@@ -271,3 +268,10 @@ governing permissions and limitations under the License.
   flex-shrink: 0;
   flex-basis: 0%;
 }
+
+.spectrum-TabsPanel-tabpanel {
+  flex-grow: 1;
+  display: flex;
+  align-items: stretch;
+  flex-direction: column;
+}
```
Comment (LFDanLu) [inline packages/@adobe/spectrum-css-temp/components/tabs/index.css:277]: Out of curiosity, did you have a use case that prompted the above css change? I figured that the user could handle the styling of the tabpanel themselves via styling the `Content` they pass

**Review by LFDanLu — COMMENTED** [review]
The collapseWrapper css change looks good to me, just had a question regarding the tabpanel change

File: packages/@adobe/spectrum-css-temp/components/tabs/index.css (hunk 1 above)
Comment (tomsontom) [reply packages/@adobe/spectrum-css-temp/components/tabs/index.css]: you are right the last 3 props are not needed - I had them in there just for convenience. The flex-grow is required although

**Review by LFDanLu — DISMISSED** [review]
LGTM

**Review by snowystinger — APPROVED** [review]
I hope you don't mind but I added a story so we'd know about this case while manually testing

**Review by LFDanLu — APPROVED** [review]

**Comment by LFDanLu** [comment]
@tomsontom Thanks for the contribution!

**Pull request merged**